flexget/plugins/sites/jackett_torznab.py

```diff
@@ -0,0 +1,141 @@
+          website: https://website
+          apikey: xxxxxxxxxxxxxxxxxxxxxxxxxx
+          category: movie
+
+    Category is any of: movie, tvsearch, search
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'category': {'type': 'string', 'enum': ['movie', 'tvsearch', 'tv', 'search']},
+            'url': {'type': 'string', 'format': 'url'},
+            'website': {'type': 'string', 'format': 'url'},
+            'apikey': {'type': 'string'}
+        },
+        'required': ['category'],
+        'additionalProperties': False
+    }
+
+    def build_config(self, config):
+        log.debug(type(config))
+
+        if config['category'] == 'tv':
+            config['category'] = 'tvsearch'
+
+        if 'url' not in config:
+            if 'apikey' in config and 'website' in config:
+                params = {
+                    't': config['category'],
+                    'apikey': config['apikey'],
+                    'extended': 1
+                }
+                config['url'] = config['website'] + '/api?' + urlencode(params)
+
+        return config
+
+    def fill_entries_for_url(self, url, task):
+        entries = []
+        log.verbose('Fetching %s' % url)
+
+        try:
+            r = task.requests.get(url)
+        except task.requests.RequestException as e:
+            log.error("Failed fetching '%s': %s" % (url, e))
+
+        rss = feedparser.parse(r.content)
+        log.debug("Raw RSS: %s" % rss)
+
+        if not len(rss.entries):
+            log.info('No results returned')
+
+        for rss_entry in rss.entries:
+            new_entry = Entry()
+
+            for key in list(rss_entry.keys()):
+                new_entry[key] = rss_entry[key]
+            new_entry['url'] = new_entry['link']
+            if rss_entry.enclosures:
+                size = int(rss_entry.enclosures[0]['length'])  # B
+                new_entry['content_size'] = old_div(size, 2 ** 20)  # MB
+            entries.append(new_entry)
+        return entries
+
+    def search(self, task, entry, config=None):
+        config = self.build_config(config)
+        if config['category'] == 'movie':
+            return self.do_search_movie(entry, task, config)
+        elif config['category'] == 'tvsearch':
+            return self.do_search_tvsearch(entry, task, config)
+        elif config['category'] == 'search':
+            return self.do_search_search(entry, task, config)
+        else:
+            entries = []
+            log.warning("Not done yet...")
+            return entries
+
+    def do_search_tvsearch(self, arg_entry, task, config=None):
+        log.info('Searching for %s' % (arg_entry['title']))
+        # normally this should be used with next_series_episodes who has provided season and episodenumber
+        if 'series_name' not in arg_entry or 'series_season' not in arg_entry or 'series_episode' not in arg_entry:
```

Comment:
proposed syntax:
` if not all(value in arg_entry for value in ['series_name', 'series_episode', 'series_season']):`
just a suggestion